test/index.test.js

```diff
@@ -354,22 +354,22 @@ describe('lmdb-js', function () {
 					should.equal(db.get('key1'), 'done!');
 				}
 			});
-      if (version.patch >= 90)
```

Comment:
Some spurious lint/write-space updates, really the RangeIterable suite is important test that was added/updated.